docs/scope.md

```diff
@@ -0,0 +1,53 @@
+# Document Purpose
+
+This document is a guide for contributors and Senate to decide if a feature is within "scope" for pokeemerald-expansion. If a feature is not in scope, then it should not be merged. Even if an opened PR is within scope, this does not mean it will be merged, as acceptance criteria will often come down to the details of the implementation.
+
+# Definitions
+
+* **Showdown Supported (SS)**: A core series game who's metagame can be played on Showdown.
+  * Notably, this is every [core series game](https://bulbapedia.bulbagarden.net/wiki/Core_series#List_of_core_series_games) except Pokémon Legends: Arceus and Pokémon Legends: Z-A.  
+* **Base Expansion Version**: A .gba file built from an unmodified `master` branch of `pokeemerald-expansion`.  
```

Comment:
Should we add a description for upcoming or tagged releases?

Reply:
I'm not sure what you mean, could you please elaborate?

Reply:
Oh I think I understand now: I don't think so, but if you suggest some, I won't fight you

Reply:
Based on the description of "A .gba file built from an unmodified `master` branch of `pokeemerald-expansion`.", it made me ask "then what would be a gba file based off upcoming?

Reply:
probably it's not needed

Reply:
Insofar as the term "base" is used in the doc, the statements should hold true for both `master` and `upcoming`. I'm equally fine to leave it as is or modify the statement to "A .gba file built from an unmodified `master` or `upcoming` branch of `pokeemerald-expansion`. "

Reply:
> Insofar as the term "base" is used in the doc, the statements should hold true for both `master` and `upcoming`. I'm equally fine to leave it as is or modify the statement to "A .gba file built from an unmodified `master` or `upcoming` branch of `pokeemerald-expansion`. "

This makes the most sense to me :)